Makefile

```diff
@@ -34,7 +34,7 @@ dev: env
 ## Runs unit tests.
 test: lint
 	@echo "Running unit tests."
-	$(PIPENV_CMD) run tox
+	$(PIPENV_CMD) run tox --recreate
```

Comment:
I think this is not necessary anymore, [PR-40](https://github.com/HewlettPackard/py-spiffe/pull/40) already added the flag and is already merge. (Title does not mention this change but is listed on the description, sorry to mix things on the same PR) 
Is my point correct or I'm missing something?

Reply:
Indeed - that change was not in `master` at the time I made this pull request, and it snuck in via unrelated changes. :-) I'll abandon this one. Thanks!

Reply:
Yes, it would have been better to add the change in a different PR. Sorry for the confusion. 
Thanks Scott